### PR TITLE
Osd23682: Fixing README file

### PR DESCRIPTION
**Pull request by faldanarh**
### What type of PR is this?

_documentation_

### What this PR does / Why we need it?

The README file is pointing to the old script name.

### Which Jira/Github issue(s) does this PR fix?

_Resolves #_https://issues.redhat.com/browse/OSD-23682

### Pre-checks (if applicable)

- [x] Validated the changes in a cluster
- [x] Included documentation changes with PR

### CODE REVIEW

File: scripts/CEE/worker-node-operations/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+- cordon: To cordon worker nodes
+- uncordon: To uncordon worker nodes
+- drain: This will cordon and then drain worker nodes
+- reboot: This will cordon, drain and then reboot worker nodes
+
+## Usage
+
+**ATTENTION** ⚠️ This script must only be used by the MCS (Managed Cloud Services) CRU (Crew Response Unit) team.
+- If you need this script to be used, contact a MCS CRU member.
```
Comment (MitaliBhalla) [inline scripts/CEE/worker-node-operations/README.md:19]:
```suggestion
- If you need this script to be used, contact a MCS CRU member.
```
Why not SRE as well?

**Review by MitaliBhalla — CHANGES_REQUESTED** [review]

File: scripts/CEE/worker-node-operations/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+```bash
+ocm backplane managedjob create CEE/worker-node-operations -p WORKER="ip_x.x.x.x" -p ACTION="drain" -p DRAINMODE="--ignore-daemonsets --delete-emptydir-data --force"
+```
+Or, to avoid the PDB check: 
+```bash
+ocm backplane managedjob create CEE/worker-node-operations -p WORKER="ip_x.x.x.x" -p ACTION="drain" -p DRAINMODE="--ignore-daemonsets --delete-emptydir-data --force --disable-eviction"
+```
+Ref: https://docs.openshift.com/container-platform/4.15/nodes/nodes/nodes-nodes-working.html
+
+
```
Comment (MitaliBhalla) [inline scripts/CEE/worker-node-operations/README.md:43]:
```suggestion
```

**Review by MitaliBhalla — CHANGES_REQUESTED** [review]

File: scripts/CEE/worker-node-operations/README.md (hunk 1 above)
Comment (faldanarh) [reply scripts/CEE/worker-node-operations/README.md]: This is only a recommendation for the MCS members (who is not part of the CRU) to have a CRU member supervision.

**Review by MitaliBhalla — APPROVED** [review]

**Pull request merged**